pkg/helm/releases.go

```diff
@@ -0,0 +1,39 @@
+package helm
+
+import (
+	"time"
+
+	helmv2beta1 "github.com/fluxcd/helm-controller/api/v2beta1"
+	sourcev1beta1 "github.com/fluxcd/source-controller/api/v1beta1"
+	pb "github.com/weaveworks/weave-gitops/pkg/api/profiles"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// MakeHelmRelease returns a HelmRelease object given a profile, cluster, namespace, and version.
+func MakeHelmRelease(p *pb.Profile, cluster, namespace string) *helmv2beta1.HelmRelease {
```

Comment:
given this is relating to profiles only, does it make sense to have it in a different package? ` pkg/services/profiles/helm/`